src/main/java/racing/domain/Car.java

```diff
@@ -0,0 +1,24 @@
+package racing.domain;
+
+import racing.util.ForwardUtil;
+
+public class Car {
+
+    private int distance;
+
+    public int getDistance() {
+        return distance;
+    }
+
+    public void move(int value)
+    {
+        this.distance += getDistance(value);
+    }
```

Comment:
자바 컨벤션에 맞지 않는 중괄호 `{}` 사용인 듯합니다. 

```suggestion
    public void move(int value) {
        this.distance += getDistance(value);
    }
```